sklearn/neighbors/tests/test_lof.py

```diff
@@ -53,8 +54,7 @@ def test_lof():
     assert_array_equal(clf.fit_predict(X), 6 * [1] + 2 * [-1])
 
 
-@ignore_warnings(category=DeprecationWarning)
-# contamination changed to 'auto' 0.22
+@pytest.mark.filterwarnings('ignore:default contamination parameter 0.1')
```

Comment:
Should we be giving the version number of scikit-learn where this line should be remove: "# 0.22" at the end of the line?